ProjectSource/src/test/java/com/kayako/sdk/ParserFactoryTest.java

```diff
@@ -0,0 +1,222 @@
+package com.kayako.sdk;
+
+import com.kayako.sdk.base.parser.DataNodeParser;
+import com.kayako.sdk.error.response.*;
+import com.kayako.sdk.helpcenter.articles.ArticleParser;
+import com.kayako.sdk.helpcenter.category.CategoryParser;
+import com.kayako.sdk.helpcenter.locale.LocaleParser;
+import com.kayako.sdk.helpcenter.search.SearchArticleParser;
+import com.kayako.sdk.helpcenter.section.SectionParser;
+import com.kayako.sdk.helpcenter.user.UserMinimalParser;
+import com.kayako.sdk.messenger.attachment.AttachmentParser;
+import com.kayako.sdk.messenger.attachment.ThumbnailParser;
+import com.kayako.sdk.messenger.conversation.ConversationParser;
+import com.kayako.sdk.messenger.conversation.fields.priority.PriorityParser;
+import com.kayako.sdk.messenger.conversation.fields.readmarker.ReadMarkerParser;
+import com.kayako.sdk.messenger.conversation.fields.status.StatusParser;
+import com.kayako.sdk.messenger.conversation.fields.team.TeamParser;
+import com.kayako.sdk.messenger.conversationstarter.ConversationStarterParser;
+import com.kayako.sdk.messenger.message.MessageParser;
+import static org.hamcrest.CoreMatchers.instanceOf;
+
+import com.kayako.sdk.messenger.rating.RatingParser;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Locale;
+
+public class ParserFactoryTest {
+
+    private Locale locale;
+
+    @Before
+    public void setup() {
+        locale = Locale.ENGLISH;
+    }
+
+    @Test
+    public void getErrorListParser() {
+        Assert.assertThat(ParserFactory.getErrorListParser(), instanceOf(ErrorListParser.class));
+    }
+
+    @Test
+    public void getErrorParser() {
+        Assert.assertThat(ParserFactory.getErrorParser(), instanceOf(ErrorListParser.class));
```

Comment:
⚠️  It could be a good idea to import the `static` assertions. This way the code becomes smaller and avoids repetition.  

**[More details](https://confluence.devfactory.com/x/MxNrEw)**
          
## The possible solution in this case

```JAVA
import static org.junit.Assert.assertThat;
...
assertThat(ParserFactory.getErrorParser(), instanceOf(ErrorListParser.class));
```
 

<a id="ca93d190-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />